src/components/Common/SkeletonLoading.tsx

```diff
@@ -0,0 +1,130 @@
+                  <Skeleton className="h-3 w-20" />
+                </div>
+              </div>
+            </td>
+            <td className="flex-0 px-6 py-4">
+              <Skeleton className="h-4 w-16" />
+            </td>
+            <td className="px-10 py-4 text-sm">
+              <Skeleton className="h-4 w-20" />
+            </td>
+            <td className="px-4 py-4 text-sm whitespace-nowrap">
+              <Skeleton className="h-4 w-24" />
+            </td>
+            <td className="px-4 py-4">
+              <Skeleton className="h-8 w-20 rounded-md" />
+            </td>
+          </tr>
+        ))}
+      </tbody>
+    </table>
+  </div>
+);
+
+const NoteSkeleton = () => (
+  <div className="p-4 rounded-lg bg-gray-100 animate-pulse">
+    <div className="flex items-start gap-3">
+      <div className="w-8 h-8 rounded-full bg-gray-200" />
+      <div className="flex-1 space-y-2">
+        <Skeleton className="h-4 w-24" />
+        <Skeleton className="h-4 w-3/4" />
+      </div>
+    </div>
+  </div>
+);
+const CardGridSkeleton = () => (
+  <Card className="h-full">
+    <CardContent className="p-4 sm:p-6">
+      <div className="flex flex-col h-full gap-4">
+        <div className="flex gap-4">
+          <Skeleton className="h-12 w-12 sm:h-16 sm:w-16 rounded-full flex-shrink-0" />
+          <div className="flex flex-col min-w-0 flex-1">
+            <div className="flex flex-col gap-1">
+              <Skeleton className="h-5 w-32 mb-1" />
+              <div className="flex items-center gap-2 flex-wrap">
+                <Skeleton className="h-4 w-20" />
+                <Skeleton className="h-4 w-16" />
+              </div>
+            </div>
+          </div>
+        </div>
+
+        <div className="grid grid-cols-2 gap-4 text-sm">
+          <div>
+            <Skeleton className="h-4 w-16 mb-1" />
+            <Skeleton className="h-4 w-24" />
+          </div>
+          <div>
+            <Skeleton className="h-4 w-24 mb-1" />
+            <Skeleton className="h-4 w-16" />
+          </div>
+        </div>
+
+        <div className="mt-auto pt-2">
+          <Skeleton className="h-8 w-full rounded-md" />
+        </div>
+      </div>
+    </CardContent>
+  </Card>
+);
+interface SkeletonLoadingProps {
+  count: number;
+  element: "CardGridSkeleton" | "TableSkeleton" | "NoteSkeleton";
+}
+
+export const SkeletonLoading = ({ count, element }: SkeletonLoadingProps) => {
+  return (
+    <>
+      {element === "CardGridSkeleton" ? (
+        Array.from({ length: count }, (_, index) => (
+          <div key={index} className="skeleton-item">
+            <CardGridSkeleton />
+          </div>
+        ))
+      ) : element === "NoteSkeleton" ? (
+        Array.from({ length: count }, (_, index) => (
+          <div key={index} className="skeleton-item">
+            <NoteSkeleton />
+          </div>
+        ))
+      ) : (
+        <TableSkeleton count={count} />
+      )}
```

Comment:
why not just export the three parent components directly instead of having a `SkeletonLoading` component?